fplll/sieve/sieve_gauss.cpp

```diff
@@ -292,16 +292,12 @@ template <class ZT, class F> void GaussSieve<ZT, F>::print_final_info()
     cout << "# [info] max(|L|)=" << max_list_size;
     cout << " log2(max|L|)/n=" << log2(max_list_size) / nc << endl;
     cout << "# [info] true max|L| = " << first_size << endl;
-    ;
     cout << "# [info] true log2(max|L|)/n = " << log2(first_size) / nc << endl;
-    cout << "# [info] sv is" << endl;
-  }
-  if (verbose)
-  {
-    final_norm.set_z(best_sqr_norm);
-    final_norm.sqrt(final_norm, GMP_RNDN);
-    cout << "# [info] |sv| = " << final_norm << " (" << best_sqr_norm << ")" << endl;
   }
+  cout << "# [info] sv is " << return_first() << endl;
```

Comment:
I have to admit, I'm not too keen on the unconditional printing here. For example, if this code is used inside fpylll then there's a bunch of spurious printed messages when calling this function in a loop.

Reply:
Right, I remembered that now. I added it since current ./latsieve binary will not output the vector found which is a bit strange.  Let me think what to do with that.

Reply:
Martin, now I enabled the verbose = True when called from command line which will output the shortest vector. When used from fpylll, one can use verbose = False to GaussSieve object to avoid the verbose problem.